plugin.yaml

```diff
@@ -40,6 +40,22 @@ node_types:
                 A dictionary of parameters allowed by docker-py to the
                 create_container function.
               default: {}
+            daemon_client:
+              description: >
+              Docker client information necessary to connect to Docker service
+                - transport_protocol
+                - localhost
+                - host_port
+                - api_version
+                - connection_timeout
+                - enable_tls
+              default:
+                transport_protocol: tcp
+                host_string: localhost
+                host_port: 2375
+                api_version: '1.19'
+                connection_timeout: 60
+                enable_tls: false
```

Comment:
I'm think that it's better to specify this once and not repeat in the plugin.yaml. Maybe this would be cleaner:
```
dsl_definitions:
    - &daemon_client_configuration
      transport_protocol: tcp
      host_string: localhost
      host_port: 2375
      api_version: '1.19'
      connection_timeout: 60
      enable_tls: false
...
node_types:
    cloudify.docker.Container:
....
         create:
           implementation: docker.docker_plugin.tasks.create_container
           inputs:
             params:
               description: >
                 A dictionary of parameters allowed by docker-py to the
                 create_container function.
               default: {}
            daemon_client:
                default: *daemon_client_configuration
```
see: http://docs.getcloudify.org/3.3.0/dsl-spec-dsl-definitions.html
For the description part, we should probably specify the description one time and then in the later repetitions say "see above". Clear?